plugins/Web/plugin.py

```diff
@@ -149,19 +149,31 @@ def noIgnore(self, irc, msg):
     def getTitle(self, irc, url, raiseErrors):
         size = conf.supybot.protocols.http.peekSize()
         timeout = self.registryValue('timeout')
-        (target, text) = utils.web.getUrlTargetAndContent(url, size=size,
+        try:
+            (target, text) = utils.web.getUrlTargetAndContent(url, size=size,
                 timeout=timeout)
+        except Exception as e:
+            if raiseErrors:
+                irc.error(_('That URL raised <' + str(e)) + '>',
+                          Raise=True)
+                return None
```

Comment:
That return is dead code

Reply:
Sorry for delay, been having really busy lately and despite being a simple change I wanted to test it anyway. Done.

Reply:
you forgot this one

Reply:
True. Fixed now.